tests/qa_automation/qa_run.pm

```diff
@@ -70,9 +70,14 @@ sub qaset_config {
         assert_script_run("chmod +x $boot_local");
     }
 
-    if (is_sle('=15') or is_sle('=15-SP1')) {
-        record_soft_failure("qaperf.service fails to start bsc#1182914");
-        assert_script_run("systemctl reset-failed qaperf.service");
+    # Reset the failed state of all units so that only new failures are recorded
+    assert_script_run("systemctl reset-failed");
```

Comment:
Don't you want to run this after mdadm is fixed ? Don't know if it matters

Reply:
No, this is really because there are already some recorded failures, eg for qaperf.service which fails because `root/qaset/config` is only available later. So instead of systemd testsuite to think those are failing, we need to reset the failed states and rerun them then.

Reply:
Also just to mention, mdadm never fails, does start, but does not remain running, so it is not a failed state per se